code_to_gast/expression/py_expression.py

```diff
@@ -48,4 +48,7 @@ def function_def_to_gast(node):
     gast["id"] = pr.node_to_gast(node.name)
     gast["params"] = pr.node_to_gast(node.args)
     gast["body"] = pr.node_to_gast(node.body)
-    return gast
\ No newline at end of file
+    return gast
+
+def subscript_to_gast(node):
+    return {"type": "subscript", "index": pr.node_to_gast(node.slice.value), "value": pr.node_to_gast(node.value)} # TODO expand to cover slices of a list
```

Comment:
Can we move the TODO above the return statement?

Reply:
Also might be easier to read if we make the dictionary multiline 
```
{
    "type": "subscript",
    ... etc
}
```

Reply:
Will do.